### PR TITLE
added support for synapse optic REST API via use_optic

### PR DESCRIPTION
**Pull request by bertdg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [https://github.com/demisto/content/issues/19204](https://github.com/demisto/content/issues/19204)

## Description
Version 1.0.1 of the Synapse content pack does not support Vertex Synapse Optic. The login url needs to be updated.
I've added an integration option called use_optic and updated the code and release notes

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
No change from v1.0.1 of Synapse integration

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bertdg** [comment]
FYI I've reviewed the CircleCI output and all of those warnings are from the current/published/certified version of this integration v1.0.1. If there is any way my PR which adds a feature to login to current versions of synapse on top of the already approved/certified integration (which is failing on the CircleCI checks). Thanks for your consideration of this request.

**Comment by ShacharKidor** [comment]
Hi @bertdg, 
Thank you for contributing!

I apologize for the delay in responding to your PR, I'll publish my review in a few days.
Thanks for waiting :)

File: Packs/Synapse/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Synapse
+- Added configuration option (use_optic) to support Vertex Synapse Optic
```
Comment (ShacharKidor) [inline Packs/Synapse/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Added the configuration parameter **Use Synapse Optic** to support Vertex Synapse Optic.
```

File: Packs/Synapse/Integrations/Synapse/Synapse.yml [hunk 2]
```diff
@@ -46,6 +46,10 @@ configuration:
   name: good_tag
   required: false
   type: 0
+- display: Use Synapse Optic
+  name: use_optic
+  required: false
+  type: 8
```
Comment (ShacharKidor) [inline Packs/Synapse/Integrations/Synapse/Synapse.yml:52]: Worth adding the 'additionalinfo' description for the 'use_optic' config parameter - similar to the description in all the other parameters.

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @bertdg , 
Thank you for contributing!

**In order to complete your contribution:** 
1. I added 2 little comments - please review them. 

2. Please updated the docker image in the integration YML to: 1.0.0.24037 - it will solve the build's failure. 

3. Let's schedule a short demo - please let me when would be a good time for you. 

Feel free to contact me with any questions :)

**Comment by ShacharKidor** [comment]
> FYI I've reviewed the CircleCI output and all of those warnings are from the current/published/certified version of this integration v1.0.1. If there is any way my PR which adds a feature to login to current versions of synapse on top of the already approved/certified integration (which is failing on the CircleCI checks). Thanks for your consideration of this request.

@bertdg - about that, I'm not sure I understood your question. Can I find you on slack?

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**